index.html

```diff
@@ -20,71 +20,110 @@
     <header class="header_1">
         <h1 class="heading_title">My Logo</h1>
         <nav class="navbar">
-            <img class="img2" src="resources/menu.png" alt="Menu Icon">
+            <ul class="item-navigation hidden_mobile">
+                <li><a href="#">Portofilio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+                <li><a href="#"><img src="resources/Icon - Mail.png" alt="Mail Icon"></a></li>
+            </ul>
+            <img class="img2 hidden_desktop" src="resources/menu.png" alt="Menu Icon">
         </nav>
     </header>
 
     <!-- the main section starts from here -->
     <main>
         <section>
             <div class="description_section">
-                <h1 class="h1_heading">Hey There.<br>
-                    I’m Alex.</h1>
-                <h3 class="h3_heading">I'm a Software Developer</h3>
-                <p class="paragraph">I can help you build a product , feature or website Look through some of my work
-                    and
-                    experience! If you like what you see and have a project you need coded,don’t hesitate to contact me.
-                </p>
-                <div>
+                <div class="headline">
+                    <h1 class="h1_heading">Hey There.<br>
+                        I’m Alex.</h1>
+                    <h3 class="h3_heading">I'm a Software Developer</h3>
+                    <p class="paragraph">I can help you build a product , feature or website Look
+                        through some of my work and experience! If you like what
+                        you see and have a project you need coded,
+                        don’t hesitate to contact me.
+                    </p>
+                </div>
+                <div class="social-media-icons">
                     <ul class="social">
                         <li class="lists"><a href="#"> <img src="./resources/logo1.png" alt="logo"> </a></li>
                         <li class="lists"><a href="#"> <img src="./resources/logo2.png" alt="logo"> </a></li>
                         <li class="lists"><a href="#"> <img src="./resources/logo3.png" alt="logo"> </a></li>
-                        <li class="lists"><a href="#"> <img src="./resources/logo4.png" alt="logo"> </a>></li>
+                        <li class="lists"><a href="#"> <img src="./resources/logo4.png" alt="logo"> </a></li>
                         <li class="lists"><a href="#"> <img src="./resources/logo5.png" alt="logo"> </a></li>
                     </ul>
                 </div>
                 <div class="image_container">
-                    <img src="resources/Illustration.png" alt="image">
+                    <img id="desktop_v_img" src="resources/Illustration-header.svg" class="hidden_mobile" alt="">
+                    <img src="resources/Illustration.png" class="hidden_desktop" alt="image">
```

Comment:
- [ ] Great job on addressing the changes suggested by the previous reviewer. However, I noticed that the `image` link in your header section appears to be broken. As a result, the `image` is not being loaded properly, which can affect the professional look and user experience of your website. To resolve this issue, please consider the following hint:

`HINT`: To specify the correct path for the image source (`src` attribute), add `./` in front of your image path. This will indicate the specific location from which the image should be fetched or looked for.

By incorporating this hint, you should be able to fix the broken image link and ensure that your header section displays the image correctly, enhancing the overall quality and user experience of your website. See below for more details:
<img width="1437" alt="Screenshot 2023-06-15 at 22 36 05" src="https://github.com/Shahid-Bagwan/Portfolio/assets/107192076/919ca6f9-207a-493f-bd8d-8b33dba51873">

Reply:
@Strangeal  I checked but this issue did not came up, i downloaded the files and then checked and still there was not a problem.
![my_result](https://github.com/Shahid-Bagwan/Portfolio/assets/112220551/7e3b1e01-0073-4c01-a070-bc65718a9076)